src/renderer/components/dialog/DeleteOpdsFeedConfirm.tsx

```diff
@@ -28,33 +32,37 @@ export class DeleteOpdsFeedConfirm extends React.Component<DeleteOpdsFeedConfirm
     }
 
     public render(): React.ReactElement<{}> {
-        const {__} = this.props;
-        if (!this.props.feed) {
-            return <></>;
+        if (!this.props.open) {
```

Comment:
Shouldn't this conditional statement also check for `!this.props.feed`?

Reply:
9688474